Clarify which values are optional. (#6346)
EnterpriseDB/docs: product_docs/docs/pem/9/registering_agent.mdx

|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `--pem-server`                  | The IP address of the PEM backend database server. This parameter is required.                                                                                                                                                                                                    |
| `--pem-port`                    | The port of the PEM backend database server. The default value is `5432`.                                                                                                                                                                                                         |

> | `--pem-user`                    | The name of a database user with `pem_admin` and `rolcreaterole` roles (or a superuser) on the PEM backend database server. This user will be used to connect to the PEM server to perform agent registration.  This parameter is required.                                                                                                                                                         |

Reviewer: Yes

> | `--pem-user`                    | The name of a database user with `pem_admin` and `rolcreaterole` roles (or a superuser) on the PEM backend database server. This user will be used to connect to the PEM server to perform agent registration.  This parameter is required.                                                                                                                                                         |

Author: ```suggestion
| `--pem-user`                    | The name of a database user with the `pem_admin` role and the `rolcreaterole` flag set (or a superuser) on the PEM backend database server. This user will be used to connect to the PEM server to perform agent registration.  This parameter is required.                                                                                                                                                         |
```

> | `--pem-agent-user`              | The name of a database user on the PEM backend database server. This user will be used by the agent after registration to open connections to the PEM database server to write probe data, evaluate alerts, etc. This parameter is optional. If omitted, the agent will connect using a new user created during registration named `agent<N>` where `<N>` is the agent ID. Note that this user is always created and even if you specify a `pem-agent-user` the agent will use `SET ROLE agent<N>` to switch roles after the connection is made. |

Reviewer 2: ```suggestion
| `--pem-agent-user`              | The name of a database user on the PEM backend database server. After registration, the agent will use this user to open connections to the PEM database server to write probe data, evaluate alerts, etc. This parameter is optional. If omitted, the agent will connect using a new user created during registration named `agent<N>` where `<N>` is the agent ID. Note that this user is always created and even if you specify a `pem-agent-user` the agent will use `SET ROLE agent<N>` to switch roles after the connection is made. |
```

| `--config-dir`                  | The directory path for the configuration file. The default is `<pemworker path>/../etc`.                                                                                                                                                                                          |
| `--display-name`                | A user-friendly name for the agent to display in the PEM browser tree. In PEM 9.6 and later, the default is the host's fully qualified domain name (FQDN), falling back to the hostname if this option isn't set. For releases earlier than PEM 9.6, the default is the hostname. |
| `--force-registration`          | Include the `force-registration` clause to register the agent with the arguments provided. This clause is useful if you're overriding an existing agent configuration. The default value is `Yes`.                                                                                |

> | `--group`                       | The name of a group inw which to place the agent. This parameter is optional, if omitted the agent will not be placed in a group.                                                                                                                                                                                                                            |

Author: ```suggestion
| `--group`                       | The name of a group in which to place the agent. This parameter is optional, if omitted the agent will not be placed in a group.                                                                                                                                                                                                                            |
```

> | `--group`                       | The name of a group inw which to place the agent. This parameter is optional, if omitted the agent will not be placed in a group.                                                                                                                                                                                                                            |

Reviewer 2: ```suggestion
| `--group`                       | The name of a group to place the agent. This parameter is optional, if omitted the agent will not be placed in a group.                                                                                                                                                                                                                            |
```

> | `--group`                       | The name of a group inw which to place the agent. This parameter is optional, if omitted the agent will not be placed in a group.                                                                                                                                                                                                                            |

Reviewer 3: there is a typo, `inw` to `in`

> | `--team`                        | The name of a database role on the PEM backend database server. Access to this agent will be restricted to only the named role, the owner, and the `pem_admin` role. This parameter is optional. If omitted, no team will be assigned meaning all users can access this agent.                                                                                                                                                                       |

Reviewer 2: ```suggestion
| `--team`                        | The name of a database role on the PEM backend database server. Access to this agent will be restricted to only the named role, the owner, and the `pem_admin` role. This parameter is optional. No team will be assigned if omitted, meaning all users can access this agent.                                                                                                                                                                       |
```

> | `--owner`                       | The name of a database user on the PEM backend database server. This user will be assigned as the owner of the agent. If omitted, the specified `pem-user` will be assigned as the owner.                                                                                                                                                                                              |

Reviewer 2: ```suggestion
| `--owner`                       | The name of a database user on the PEM backend database server. This user will be assigned as the owner of the agent. The specified `pem-user` will be assigned as the owner if omitted.                                                                                                                                                                                              |
```

| `--enable-heartbeat-connection` | Enable the `enable-heartbeat-connection` parameter to create a dedicated heartbeat connection between the PEM agent and server to update the active status. The default value is `False`.                                                                                         |
| `--enable-smtp`                 | Enable the `enable-smtp parameter` to allow the PEM agent to send the email on behalf of the PEM server. The default value is `False`.                                                                                                                                            |
| `--enable-snmp`                 | Enable the `enable-snmp parameter` to allow the PEM agent to send the SNMP traps on behalf of the PEM server. The default value is `False`.                                                                                                                                       |

> | `-o`                            | Used to override the configuration file options. See below for example usage.                                                                                                                                                                                                            |

Reviewer 2: ```suggestion
| `-o`                            | Used to override the configuration file options. See the below [example](#overriding-default-configurations---examples) for usage.                                                                                                                                                                                                            |
```
